osu.Game/Overlays/Chat/ChannelList/ChannelListItem.cs

```diff
@@ -160,6 +161,20 @@ protected override void OnHoverLost(HoverLostEvent e)
             };
         }
 
+        protected override bool OnMouseDown(MouseDownEvent e)
```

Comment:
Old design implementation had `OnMouseUp()` and `button.TriggerClick()` for the clicking sound:
https://github.com/ppy/osu/blob/f53593553d6776e87010d034acecc9e057c75fad/osu.Game/Overlays/Chat/Tabs/ChannelTabItem.cs#L151-L159

Also browsers (e.g. Safari, Firefox) do on mouse up.

Reply:
I've updated the code to user the simpler version as per the old implementation, but for now I'm leaving on mouse down as I think it feels better.